torchvision/datasets/sstem.py

```diff
@@ -0,0 +1,134 @@
+from __future__ import print_function
+
+import os
+
+from PIL import Image
+import skimage
```

Comment:
This would make `torchvision` dependent on `skimage` at import. Better do a lazy `import` as you suggested.